dask/dataframe/rolling.py

```diff
@@ -391,20 +392,28 @@ def quantile(self, quantile):
         return self._call_method("quantile", quantile)
 
     @derived_from(pd_Rolling)
-    def apply(self, func, args=(), kwargs={}, **kwds):
-        # TODO: In a future version of pandas this will change to
-        # raw=False. Think about inspecting the function signature and setting
-        # to that?
-        if PANDAS_VERSION >= "0.23.0":
-            kwds.setdefault("raw", None)
-        else:
-            if kwargs:
-                msg = (
-                    "Invalid argument to 'apply'. Keyword arguments "
-                    "should be given as a dict to the 'kwargs' argument. "
-                )
-                raise TypeError(msg)
-        return self._call_method("apply", func, args=args, kwargs=kwargs, **kwds)
+    def apply(
```

Comment:
This is reworked a bit. May warrant a bit of special attention.

1. Adds support for `engine='numba'`.
2. Makes the default value for `raw` depend on the pandas version.

Reply:
This looks fine to me.  cc'ing @kkraus14 just because I think that cudf might care about this API point.

Reply:
I don't think this is currently working as it is as we don't support the `args` or `kwargs` parameters, but I'm concerned the use of `PANDAS_GT_100` doesn't account for whether the partitions are Pandas or cuDF and basically forces us to play nice with whatever Pandas version the user has installed.

Reply:
Maybe we should ask something like `haskeyword(self._meta, "engine")` instead?

Reply:
Mmm OK. I'll change to use `has_keyword`.

This unfortunately makes future cleanups harder, since it's harder to spot that pandas>=1.0 has the engine keyword, but that's probably unavoidable without a bunch of additional indirection.